lib/chat_web/live/main_live/layout/uploader.ex

```diff
@@ -223,14 +272,31 @@ defmodule ChatWeb.MainLive.Layout.Uploader do
       phx-drop-target={@config.ref}
     >
       <.live_file_input
-        class="file-input block p-2 flex flex-col items-center text-sm text-black/50 file:mr-4 file:py-2 file:px-4 file:rounded-full file:border-0 file:text-sm file:font-semibold file:bg-purple file:text-purple50 file:cursor-pointer"
+        class="file-input hidden p-2 flex flex-col items-center text-sm text-black/50 file:mr-4 file:py-2 file:px-4 file:rounded-full file:border-0 file:text-sm file:font-semibold file:bg-purple file:text-purple50 file:cursor-pointer"
         upload={@config}
       />
 
       <%= if @operating_system == "Android" do %>
+        <div class="flex flex-row justify-around">
+          <a
+            class="flex justify-center items-center h-11 px-10 cursor-pointer rounded-md bg-white hover:bg-white/50"
+            phx-click={JS.dispatch("click", to: "#uploader-file-form .file-input")}
+          >
+            <.icon id="document" class="w-4 h-4 fill-grayscale" />
+            <span class="ml-2">File</span>
+          </a>
+          <a
+            class="flex justify-center items-center h-11 px-10 cursor-pointer rounded-md bg-white hover:bg-white/50"
+            phx-click={JS.dispatch("click", to: "#uploader-file-form .image-input")}
+          >
+            <.icon id="image" class="w-4 h-4 fill-grayscale" />
+            <span class="ml-2">Image</span>
+          </a>
+        </div>
+
         <input
-          accept="audio/*,image/*,video/*"
-          class="block p-2 flex flex-col items-center text-sm text-black/50 file:mr-4 file:py-2 file:px-4 file:rounded-full file:border-0 file:text-sm file:font-semibold file:bg-purple file:text-purple50 file:cursor-pointer"
+          accept="image/*"
```

Comment:
can we keep `video/*` here?  So it will show Android gallery w/ images and videos

Reply:
Should I also add `audio/*` and change the button text from "Image" to "Media"?
https://github.com/Buckitup-chat/chat/pull/157#pullrequestreview-1285443640

Reply:
I support audio and "media" title

Reply:
If it opens gallery on Android then add
@Rus9688 said that `audio/*` breaks gallery behaviuor on Android

Reply:
That's right. I've just tested it and even putting `image/*,video/*` didn't work, nor `.jpg,.jpeg,.png,.mp4`. The only way it opened gallery directly for me was `image/*`, so I won't change this in this PR. Let's experiment with that in #157.